scripts/include/lib-translations.inc.php

```diff
@@ -87,6 +87,11 @@ function is_translatable ($filename) {
 	if (in_array(basename($filename), $files_not_translated)) {
 		return false;
 	}
+
+	// exclude filenames like htmlentities.xml
+	if (preg_match("/^entities./", basename($filename))) {
+		return false;
+	}
```

Comment:
This doesn't make sense ``htmlentities.xml`` should be marked as translatable.

Reply:
Fixed `is_translatable('htmlentities.xml')` returning `false`. should return `true`

Reply:
Okay, so the *comment* makes no sense